packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -4161,4 +4161,64 @@ void main() {
 
     expect(tester.getTopLeft(find.text(hintText)).dy, topPosition);
   });
+
+  testWidgets('InputDecorator label width isn\'t affected by prefix or suffix', (WidgetTester tester) async {
```

Comment:
nit: use double quotes so you don't have to escape the `'` (I thought it was a lint rule?)